ledger-core/v2/README.md

```diff
@@ -95,6 +95,19 @@ When the pulse watcher says `INSOLAR STATE: READY`, run the following command in
              It can be changed in configuration.
 
 
+Functional test can be executed in kubernetes cluster. You need to have `kubectl` installed.
+_This will install nginx ingress in your cluster and user 80/443 port_
+
+   ```
+    make docker_build           # builds new images
+    make test_func_kubernetes   # starts insolar network and run tests
+    make make kube_stop_net     # stops network
+   ```
+To run from IDE set the environment to
+```
+INSOLAR_FUNC_RPC_URL_PUBLIC=http://localhost/api/rpc;INSOLAR_FUNC_RPC_URL=http://localhost/admin-api/rpc;INSOLAR_FUNC_KEYS_PATH=/tmp/configs/;INSOLAR_FUNC_TESTWALLET_HOST=localhost
+```
+
```

Comment:
Please add info, how to check network: `kubectl -n insolar logs --tail=20 services/pulsewatcher`

Reply:
Pulsewatcher from launchnet works wrong with kubernetes, as I remember, maybe it's better to mention it.

Reply:
added to readme